### PR TITLE
Merged in new indigo changes to bosch-devel branch

### PR DESCRIPTION
**Pull request by sosentos**
(upon merge the repo can be deleted)

### CODE REVIEW

File: amcl/src/amcl_node.cpp [hunk 1]
```diff
@@ -307,20 +279,6 @@ AmclNode::AmclNode() :
...
 
-  private_nh_.param("init_global", init_global_, false);
-
-  private_nh_.param("draw_weight_as_height", draw_weight_as_height_, false);
-
-  private_nh_.param("use_cov_from_params", use_cov_from_params_, false);
-
-  private_nh_.param("use_tf_to_update_initial_pose", use_tf_to_update_initial_pose_, true);
-
-  if(use_cov_from_params_){
```
Comment (KaijenHsiao) [inline amcl/src/amcl_node.cpp]: Hrm.  It looks like this pull request is deleting all of our additions that aren't in the ros nav stack.

**Comment by sosentos** [comment]
Urgh I did it the wrong way order wise so all of the automatic stuff will have removed our stuff.

I will close this pull request and submit a different one.

**Comment by sosentos** [comment]
Ok that was my fault.  I got rid of our changes accidentally when trying to figure out a compile error.  Now the compile error is back.  I will work on this and commit when it is solved.

**Comment by KaijenHsiao** [comment]
OK, compile is fixed and seems to work fine.

**Pull request merged**